creatures.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+$db = new PDO('mysql:host=db; dbname=AnimeCreatures', 'root', 'password');
+
+function display($db)
+{
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    $query = $db->prepare('SELECT `name`, `movie`, `year`, `special_abilities`, `creepiness`, `img_dir` FROM `creatures`;');
+    $query->execute();
+    $collection = $query->fetchAll();
+    foreach ($collection as $creature) {
+        echo "<div>"
```

Comment:
As this is inside a function, it would be better to return the string rather than echo it. That way you can unit test the function!